src/Logging/src/Abstractions/StructuredMessageProcessingLogger.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Logging;
+
+namespace Steeltoe.Extensions.Logging;
+
+public class StructuredMessageProcessingLogger : MessageProcessingLogger
+{
+    public StructuredMessageProcessingLogger(ILogger iLogger, IEnumerable<IDynamicMessageProcessor> messageProcessors = null)
```

Comment:
Please try to avoid the use of optional parameters in publicly exposed methods. They cause backwards compatibility issues.